steps/compliance/mrn-date/MRNDate.js

```diff
@@ -18,22 +18,29 @@ class MRNDate extends Question {
 
     get form() {
 
-        return form(
-
-            textField('day').joi(
-                this.content.fields.day.error.required,
-                Joi.string().regex(numbers).required()),
-
-            textField('month').joi(
-                this.content.fields.month.error.required,
-                Joi.string().regex(numbers).required()
-            ),
-
-            textField('year').joi(
-                this.content.fields.year.error.required,
-                Joi.string().regex(numbers).required()
+        const fields = this.content.fields;
+
+        return form({
+            mrnDate: convert(
+                d => DateUtils.createMoment(d.day, d.month, d.year),
+                date.required({
+                    allRequired: fields.date.error.allRequired,
+                    dayRequired: fields.date.error.dayRequired,
+                    monthRequired: fields.date.error.monthRequired,
+                    yearRequired: fields.date.error.yearRequired
+                })
+            ).check(
+                fields.date.error.invalid,
+                value => DateUtils.isDateValid(value)
+            ).check(
+                fields.date.error.future,
+                value => DateUtils.isDateInPast(value)
+            ).check(
+                fields.date.error.dateSameAsImage,
+                value => DateUtils.mrnDateDifferentToImage(value)
```

Comment:
It's good practice to always use equality first, then inequality. For example, both `isDateValid()` and `isDateInPast()` are equality (i.e. if true), however, `mrnDateDifferentToImage()` is inequality (i.e. not true). For consistency and readability can we change `mrnDateDifferentToImage()` to `mrnDateIsSameAsImage` so it reads like `value => !DateUtils.mrnDateIsSameAsImage(value)`.